source/client/output_formatter_impl.cc

```diff
@@ -288,45 +287,58 @@ std::string FortioOutputFormatterImpl::formatProto(const nighthawk::client::Outp
   fortio_output.set_numthreads(number_of_connections);
 
   // Get the result that represents all workers (global)
-  const auto& nh_global_result = this->getGlobalResult(output);
+  const auto& nh_global_result = getGlobalResult(output);
 
   // Fill in the actual QPS based on the counters
-  const auto& nh_rq_counter = this->getCounterByName(nh_global_result, "upstream_rq_total");
-  const double actual_qps = static_cast<double>(
-      nh_rq_counter.value() / std::chrono::duration<double>(actual_duration).count());
-  fortio_output.set_actualqps(actual_qps);
+  if (!visitCounter(
```

Comment:
We only throw an error about a missing counter for `upstream_rq_total`, but no others. Is there something that guarantees the other counters will always be present?

Reply:
This was addressed in another thread.